src/libraries/restore/runtime/runtime.depproj

```diff
@@ -50,7 +50,12 @@
       <UseHardlink Condition="'$(_runtimeOSFamily)' == 'FreeBSD'">false</UseHardlink>
     </PropertyGroup>
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'">
+      <HostFxFile Include="@(StoreTestHostFiles)" Condition="'%(StoreTestHostFiles.Filename)' == '$(HostFxrFileName)'" />
+      <DotnetExe Include="@(StoreTestHostFiles)" Condition="'%(StoreTestHostFiles.Filename)' == 'dotnet'" />
```

Comment:
This target now has a dependency on `OverrideRuntimeMono` but it is not hooked up correctly with a `dependsOnTargets` both targets are hooked up with a `AfterTargets` to the same target, you're just getting lucky that `OverrideRuntimeMono` is running before this target. I think we should just leave this target as is and then add a `<ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="'$(RuntimeFlavor)' == 'Mono'"` /> in this target.

Reply:
done